### ISSUE TITLE
Facilities: Duplicate IDs on accordions after component update

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#14748]
## Describe the defect
Related slack thread: https://dsva.slack.com/archives/C03LFSPGV16/p1691690032275419

There was a DST update to the <va-accordion> component done recently to allow for deep linking. Randi and PW had been working on this and these are the related tickets:
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11354
- https://github.com/department-of-veterans-affairs/content-build/pull/1650
- https://github.com/department-of-veterans-affairs/component-library/pull/782

However, after this update I now have duplicate IDs on accordions on quite a few pages.
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/de88778f-4c22-4018-b7cb-8126e5453480)

This is on both Facilities pages and PW pages:

## Example pages
**Pages that have the dup ID issue**
- [PW] https://www.va.gov/health-care/about-va-health-benefits/
- [PW] https://www.va.gov/resources/the-pact-act-and-your-va-benefits
- https://www.va.gov/south-texas-health-care/locations/north-central-federal-va-clinic/

**Pages that have accordions but do NOT have the issue**
- https://www.va.gov/boston-vet-center/
- https://www.va.gov/salt-lake-city-health-care/programs/horses-helping-veterans
- [PW] https://www.va.gov/health-care/get-health-id-card/

## To Reproduce
1. Go to one of the pages above that I indicated has the issue
2. Scroll down to where there are accordions
3. Open the inspector and inspect an accordion element
4. Note that the `<va-accordion>` element and the `<h3>` element have the same ID

## AC / Expected behavior
- [ ] In Facilities templates, the duplicate ID is resolved (ID is changed or one is removed)
- [ ] If we determine this is a global issue and that PW will be fixed by our changes, update #14747  to let PW know

## Screenshots
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/835fa519-775e-411d-9f1b-a93a92d9f536)

**Comment by laflannery** [comment]
[Public Websites issue](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14747)

**Comment by randimays** [comment]
Investigation complete! https://github.com/department-of-veterans-affairs/content-build/pull/1660

This change should also address any issues on the Facilities side.
